flink-ml-parent/flink-ml-lib/src/main/resources/META-INF/licenses/LICENSE.core

```diff
@@ -0,0 +1,49 @@
+Copyright (c) 2013 Samuel Halliday
```

Comment:
I am not familiar with the naming convention of the license notice. 
but seems to me `core` is not a very meaningful name of the package we depend on. all other `LICENSE.*` comes with the package short name like `LICENSE.jpython`, `LICENSE.protobuf` ...

Reply:
also maybe someone who's familiar with this can take a second look.

Reply:
The license is actually following the artifact id. There is something like `LICENSE.asm` in the flink-filesystem component. So I suppose this conforms to the naming convention?